department-of-veterans-affairs/va.gov-cms
# VACMS-3131: Hide EWA block on help pages.

## Description

See #3131 . 

## Testing done

Manual

## QA

1. As anon or any role, go to http://pr3135.ci.cms.va.gov/help, you shouldn't see the Entity meta blcok
1. go to any subpage, same should be true. 

![CMS_help_center___Veterans_Affairs](https://user-images.githubusercontent.com/643678/95389928-8bf99780-08c2-11eb-98e1-657cdc6bc5e6.jpg)
